lib/web/apiserver.go

```diff
@@ -1000,7 +1019,10 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/scripts/integrations/configure/deployservice-iam.sh", h.WithLimiter(h.awsOIDCConfigureDeployServiceIAM))
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/ec2", h.WithClusterAuth(h.awsOIDCListEC2))
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/eksclusters", h.WithClusterAuth(h.awsOIDCListEKSClusters))
+	// TODO(kimlisa): DELETE IN 19.0 - replaced by /v2/webapi/sites/:site/integrations/aws-oidc/:name/enrolleksclusters
+	// MUST delete with related code found in web/packages/teleport/src/services/integrations/integrations.ts(enrollEksClusters)
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/enrolleksclusters", h.WithClusterAuth(h.awsOIDCEnrollEKSClusters))
+	h.POST("/v2/webapi/sites/:site/integrations/aws-oidc/:name/enrolleksclusters", h.WithClusterAuth(h.awsOIDCEnrollEKSClusters))
```

Comment:
nit: from endpoint organization perspective - Instead of mixing `v1` and `v2` endpoints here, what do you think of creating v2 endpoint method `func (h *Handler) bindV2Endpoints() {}`? See https://github.com/gravitational/teleport/blob/master/lib/web/apiserver.go#L758 for current `bindDefaultEndpoints` method for reference.

I do not have a strong preference either way but feel like separation will make it more cleaner as number of v2 grows.

Reply:
i'll leave it as is, b/c we tend to group similar endpoints together and i think the binding method will separate them